recipes/hictk/all/conanfile.py

```diff
@@ -51,18 +51,18 @@ def config_options(self):
 
     def requirements(self):
```

Comment:
In [paulsengroup/hictk@`v1.0.0...v2.0.1`#diff-2def7ae0f4](https://github.com/paulsengroup/hictk/compare/v1.0.0...v2.0.1#diff-2def7ae0f4b6cd09f0419134e3c48c36ea67c113ff8f3a58cbc56784815f45fe) I see that a few new dependencies were added - are they missing here on pourpose?

Reply:
Correct, they are left out on purpose :).

The reason is that the hictk repo hosts code for the C++ library (which is what is currently packaged by Conan) and a CLI application (and tests of course).
The dependencies omitted here are required by either the CLI application or the tests.

I have this command that I use to check I have not forgot to add some deps:

```console
user@dev:/tmp$ find src/libhictk -type f -name "CMakeLists.txt" -exec  grep 'find_package' {} + | cut -f 2 -d : | sort -u

  find_package(Arrow REQUIRED QUIET)
find_package(bshoshany-thread-pool REQUIRED)
find_package(concurrentqueue REQUIRED)
  find_package(Eigen3 QUIET REQUIRED)
  find_package(Eigen3 REQUIRED QUIET)
find_package(FastFloat REQUIRED)
find_package(FMT REQUIRED)
find_package(HDF5 REQUIRED QUIET COMPONENTS C)
find_package(HighFive REQUIRED)
find_package(libdeflate REQUIRED)
find_package(phmap REQUIRED)
find_package(readerwriterqueue REQUIRED)
find_package(span-lite REQUIRED)
find_package(spdlog REQUIRED)
find_package(spdlog REQUIRED QUIET)
find_package(zstd REQUIRED)
```